referenceframe/frame_json.go

```diff
@@ -70,6 +71,12 @@ func NewLinkConfig(frame staticFrame) (*LinkConfig, error) {
 
 // ParseConfig converts a LinkConfig into a staticFrame.
 func (cfg *LinkConfig) ParseConfig() (*LinkInFrame, error) {
+	if cfg.ID == World {
```

Comment:
Will this break when receiving the framesystem from a remote robot's FS? Can we test this?

Reply:
I think that's covered by https://github.com/viamrobotics/rdk/pull/2415/files#diff-0b975da6011656cd3f2caf0435023573bdb5ac7c6a425e7917dd5cc9d6be63e4R794